### PR TITLE
Add error message Parsers.scala:1620 missing type keyword

### PR DESCRIPTION
**Pull request by m-sp**

### CODE REVIEW

**Comment by m-sp** [comment]
signed the CLA

**Comment by felixmulder** [comment]
Hi @m-sp - thanks for taking the time to make this PR, we'll try to get to it as soon as possible. I also think it would be best if @smarter reviewed this.

Cheers,
Felix

File: src/dotty/tools/dotc/reporting/diagnostic/messages.scala [hunk 1]
```diff
@@ -677,5 +677,18 @@ object messages {
            |
            |$fixedVarInAlternative""".stripMargin
     }
+  }  
+    
+  case class TypeParamsTypeExpected(mods:Modifiers, identifier:TermName)(implicit ctx:Context) extends Message(24) {
```
Comment (smarter) [inline src/dotty/tools/dotc/reporting/diagnostic/messages.scala]: We should always add a space between `:` and a type signature, e.g. `mods: Modifiers`

**Comment by m-sp** [comment]
thanks for the review @smarter, i addressed your comments

**Comment by felixmulder** [comment]
@m-sp: thanks for the contribution! :tada:

**Pull request merged**